ocs_ci/ocs/resources/pod.py

```diff
@@ -1779,14 +1785,18 @@ def delete_osd_removal_job(osd_id):
         bool: True, if the ocs-osd-removal job deleted successfully. False, otherwise
 
     """
-    osd_removal_job = get_job_obj(
-        f"ocs-osd-removal-{osd_id}", namespace=defaults.ROOK_CLUSTER_NAMESPACE
-    )
+    ocp_version = float(get_ocp_version())
+    if ocp_version >= 4.7:
+        job_name = "ocs-osd-removal-job"
+    else:
```

Comment:
We need a verification on ocs 4.6 build as will to make sure everything is working fine

Reply:
Okay, I will run a PR validation also on OCS 4.6